common/djangoapps/course_modes/models.py

```diff
@@ -692,3 +697,20 @@ class CourseModesArchive(models.Model):
     expiration_date = models.DateField(default=None, null=True, blank=True)
 
     expiration_datetime = models.DateTimeField(default=None, null=True, blank=True)
+
+
+class CourseModeExpirationConfig(ConfigurationModel):
+    """
+    Configuration for time period from end of course to auto-expire a course mode.
+    """
+    verification_window = models.DurationField(
+        # default to 10 days
+        default=864000000000,
+        help_text=_(
+            "The time period (in microseconds) before a course ends in which a course mode will expire"
```

Comment:
You also might want to check in the Django admin to see what the form widget is for this field -- this doc probably shouldn't mention microseconds.